symbolic-demangle/src/lib.rs

```diff
@@ -189,12 +189,22 @@ fn strip_hash_suffix(ident: &str) -> &str {
     if len < 33 {
```

Comment:
nit: you could probably remove the `else` indentation if you just returned in this if block

Reply:
clippy beat you to this one 😄